notebooks/Read_NectarCAM_events.ipynb

```diff
@@ -19,7 +19,7 @@
     "from matplotlib import pyplot as plt\n",
```

Comment:
<p><code>camera</code>  is defined beforehand in the notebook and is redefined here. Would be good to delete the first definition to avoid confusion.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/nectarchain/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>